src/store/actions/exampleActions.js

```diff
@@ -0,0 +1,8 @@
+export const addAction = () => ({
+    type: 'ADD_ACTION'
+});
+
+export const multipleAction = by => ({
+    type: 'MULTIPLE_ACTION',
+    by
```

Comment:
it's ok for now, but please remember about reading the redux docs ;) if you will have more than one data to pass, store them in one actio property called `payload`. In this example, let's stay as is